apps/ui/src/helpers/utils.ts

```diff
@@ -267,36 +268,39 @@ export function abiToDefinition(abi, chainId?: ChainId) {
     additionalProperties: false,
     properties: {}
   };
-  abi.inputs.forEach(input => {
-    definition.properties[input.name] = {};
-    definition.required.push(input.name);
+
+  abi.inputs.forEach((input, i) => {
+    const inputName = input.name ?? `input${i + 1}`;
```

Comment:
Can we do like this instead?
```suggestion
    const inputName = input.name ?? `Input ${i + 1}`;
```

Reply:
That should be fine. Updated.